docs/content/preview/migrate/yb-voyager/performance.md

```diff
@@ -10,6 +10,109 @@ menu:
+
+{{< note title="Note" >}}
+
+yb-voyager disables all the constraints only in internal sessions to migrate data.
+
+{{< /note >}}
+
+- Disable Triggers - Triggers are disabled in those sessions where yb-voyager uses to import data in a YugabyteDB cluster, because it results in repeating trigger actions in the new target database which is not required.
+
+## Other Factors
+
+There are other factors which can impact the speed of data ingestion. One or more or all of the following can be done to increase the ingestion performance.
+
+- Data load parallelism - The yb-voyager engine executes ‘N’ parallel batch ingestion jobs at any given time where ‘N’ is equal to the number of nodes in the YugabyteDB cluster. Normally this is considered as a good default practice. However, the target YugabyteDB cluster can be running on high resource machines. Machines which have a large number of cpu cores. In such cases, the default parallelism may seem unfit resulting in underutilization of cpu resources.
+
+It is good practice to have at least ‘C’ number of batch ingestion jobs run simultaneously, where ‘C’ is equal to the sum total of all the cores available in the entire cluster.
+
+{{< note title="Note" >}}
+
+If the cpu usage is going beyond 80%, then it is recommended to lower the parallelism a bit. Similarly if the cpu% is still low, one can increase the parallelism. Include the [–parallel-jobs](../../yb-voyager/yb-voyager-cli/#parallel-jobs) to the import data command to override the default parallelism by specifying the number of connections.
+
+{{< /note >}}
+
+- Batch size - Choosing a very small value of [--batch-size](../../yb-voyager/yb-voyager-cli/#batch-size) (Default is 100000) might result in slow performance, because the time spent on actual data import might be comparable/less to time spent on other tasks such as bookkeeping, setting up the client connection and so on.
+
+- Disk write contention - YugabyteDB servers can be passed with one or multiple disk volumes to store tablet data. If all the tablets are writing to a single disk, then write contention can slow down the ingestion speed. In such cases, configuring the [yb-tservers](../../../reference/configuration/yb-tserver/) with multiple disks helps in reducing the disk write contention; thereby increasing the throughput. Additionally, disks with higher IOPS and better throughput improve the write performance.
+
+- Number of table splits:
+
+  - For larger tables and indexes that are [hash](../../../architecture/docdb-sharding/sharding/#hash-sharding) sharded, specify the number of initial tablet splits desired as a part of the DDL statement of the table. This can be very beneficial to distribute the data of the table across multiple nodes right from the get go. Refer to [hash-sharded tables](../../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example to specify the number of tablets at table creation time.
```

Comment:
```suggestion
  - For larger tables and indexes that are [hash](../../../architecture/docdb-sharding/sharding/#hash-sharding) sharded, specify the number of initial tablet splits as a part of the table DDL statement. This can help distribute the table data across multiple nodes right from the get go. Refer to [hash-sharded tables](../../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example of how to specify the number of tablets at table creation time.
```